### PR TITLE
feat: add `changeActiveView` event

### PR DESCRIPTION
**Pull request by crookedneighbor**
### Summary

Adds an event to expose when the active view changes.

### Checklist

- [x] Added a changelog entry

- @crookedneighbor 
- @jplukarski

### CODE REVIEW

File: src/views/main-view.js [hunk 1]
```diff
@@ -132,7 +132,8 @@ MainView.prototype._initialize = function () {
   this._sendToDefaultView();
 };
 
-MainView.prototype._onChangeActivePaymentMethodView = function (id) {
+MainView.prototype._onChangeActivePaymentMethodView = function (data) {
```
Comment (hollabaq86) [inline src/views/main-view.js:135]: Shouldn’t this get changed to ‘onChangeActiveView’ too?

File: src/dropin.js [hunk 2]
```diff
@@ -301,6 +316,28 @@ HAS_RAW_PAYMENT_DATA[constants.paymentMethodTypes.applePay] = true;
...
+ * @typedef {object} Dropin~changeActiveView
+ * @description The event payload sent from {@link Dropin#on|`on`} with the {@link Dropin#event:changeActiveView|`changeActiveView`} event.
+ * @property {string} previousViewId The id for the previously active view. Possible values are:
+ * * `card` - The credit card form view
+ * * `paypal` - The PayPal view
+ * * `payapCredit` - The PayPal Credit view
+ * * `venmo` - The Venmo View
+ * * `googlePay` - The Google Pay view
+ * * `applePay` - The Apple Pay view
+ * * `methods` - The view presenting the avaialble payment methods (already vaulted or tokenized payment methods)
```
Comment (hollabaq86) [inline src/dropin.js:335]:
```suggestion
 * * `methods` - The view presenting the available payment methods (already vaulted or tokenized payment methods)
```

File: src/dropin.js [hunk 3]
```diff
@@ -301,6 +316,28 @@ HAS_RAW_PAYMENT_DATA[constants.paymentMethodTypes.applePay] = true;
...
+ * * `options` - The view presenting the avaialble payment options (where the customer chooses what payment method option to use). Note, if both the methods view and the options view are presented at the same time, `methods` will be shown as the view id.
```
Comment (hollabaq86) [inline src/dropin.js:336]:
```suggestion
 * * `options` - The view presenting the available payment options (where the customer chooses what payment method option to use). Note, if both the methods view and the options view are presented at the same time, `methods` will be shown as the view id.
```

File: src/views/main-view.js (hunk 1 above)
Comment (crookedneighbor) [reply src/views/main-view.js:135]: Yep!

**Review by billwerges — APPROVED** [review]

**Review by sshropshire — APPROVED** [review]

File: src/dropin-model.js [hunk 4]
```diff
@@ -235,7 +240,7 @@ DropinModel.prototype.getActivePaymentMethod = function () {
 };
 
 DropinModel.prototype.getActivePaymentView = function () {
```
[the remark this answers is not in the document]
Comment (sshropshire) [reply src/dropin-model.js:242]: +1

**Pull request merged**